### ISSUE TITLE
Recycleapp: no information as from 23 January 2024

### ISSUE THREAD

**Issue by XalaTheShepard** [pippyn/Home-Assistant-Sensor-Afvalbeheer#447]
After the latest update of Afvalbeheer, I noticed no information is available any longer for Recycleapp. Installing an older version of the integration does not solve the issue, so the issue is most likely situated at the API of Recycleapp itself.

**Comment by pippyn** [comment]
Does the website work for you?

**Comment by kimcornelis** [comment]
I've the same issue as described by XalaTheShepard but the Recycleapp website itself still provides correct information.

**Comment by XalaTheShepard** [comment]
Both the website and the smartphone app still work

**Comment by XalaTheShepard** [comment]
Sorry for the late response, but just made it home. In the logs I only find this issue: 

This error originated from a custom integration.

Logger: custom_components.afvalbeheer.API
Source: custom_components/afvalbeheer/API.py:1126
Integration: Afvalbeheer (documentation)
First occurred: 1:54:04 PM (1 occurrences)
Last logged: 1:54:04 PM

Invalid response from server for accessToken

I am not sure whether it is related to the issue. 

Kind regards, 

Robin

**Comment by pippyn** [comment]
Could you please try v5.2.16?

**Comment by Freespirit297** [comment]
v5.2.16 works with the new x-secret.

**Comment by kimcornelis** [comment]
Indeed, everything working again after installing v5.2.16. Thanks for fixing this so quickly.

**Comment by JeePee1982** [comment]
MijnAfvalwijzer still doesn't work with v5.2.16.

**Comment by pippyn** [comment]
@JeePee1982 This issue is about RecycleApp, not MijnAfvalwijzer. Please make a new issue with your config and an address to test with

**Comment by pippyn** [comment]
Can we close this issue?

**Comment by XalaTheShepard** [comment]
Also solved for me. Many thanks for the quick support 🙂
